src/frontend/komponenter/Fagsak/Vedtak/TrekkILøpendeUtbetalingNy/TrekkILøpendeUtbetaling.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useEffect, useState } from 'react';
+
+import styled from 'styled-components';
+
+import { AddCircle } from '@navikt/ds-icons';
+import { Button, Heading, Table } from '@navikt/ds-react';
+
+import type { IRestTrekkILøpendeUtbetaling } from '../../../../typer/eøs-trekk-i-løpende-ytelse';
+import NyFeilutbetaltValutaPeriode from './NyFeilutbetaltValutaPeriode';
+import TrekkILøpendeUtbetalingListeElement from './TrekkILøpendeUtbetalingListeElement';
+
+interface ITrekkILøpendeUtbetaling {
+    behandlingId: number;
+    trekkILøpendeUtbetalingListe: IRestTrekkILøpendeUtbetaling[];
+    settErUlagretNyTrekkILøpendeUtbetaling: (erUlagretNyTrekkILøpendeUtbetaling: boolean) => void;
+    erLeservisning: boolean;
+    settVisTrekkILøpendeUtbetaling: (visTrekkILøpendeUtbetaling: boolean) => void;
+}
+
+const FlexColumnDiv = styled.div`
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    margin-bottom: 3.5rem;
+    margin-top: 2rem;
+`;
+
+const FlexRowDiv = styled.div`
+    display: flex;
+    justify-content: space-between;
+`;
+
+const TrekkILøpendeUtbetaling: React.FC<ITrekkILøpendeUtbetaling> = ({
+    trekkILøpendeUtbetalingListe,
+    settErUlagretNyTrekkILøpendeUtbetaling,
+    erLeservisning,
+    settVisTrekkILøpendeUtbetaling,
+    behandlingId,
+}) => {
+    const [ønskerÅLeggeTilNyPeriode, settØnskerÅLeggeTilNyPeriode] = useState(
+        !trekkILøpendeUtbetalingListe
+    );
+
+    useEffect(() => {
+        settErUlagretNyTrekkILøpendeUtbetaling(ønskerÅLeggeTilNyPeriode);
+    }, [ønskerÅLeggeTilNyPeriode]);
+
+    if (trekkILøpendeUtbetalingListe.length === 0 && !ønskerÅLeggeTilNyPeriode) {
+        settVisTrekkILøpendeUtbetaling(false);
+    }
+
+    return (
+        <FlexColumnDiv>
+            <Heading level="2" size="small" spacing>
+                Feilutbetalt valuta
+            </Heading>
+            <Table size="small">
+                <Table.Header>
+                    <Table.Row>
+                        <Table.HeaderCell />
+                        <Table.HeaderCell scope="col">Periode</Table.HeaderCell>
+                        <Table.HeaderCell align="right" scope="col">
+                            Feilutbetalt beløp
+                        </Table.HeaderCell>
+                        <Table.HeaderCell scope="col" />
+                    </Table.Row>
+                </Table.Header>
+                <Table.Body>
+                    {trekkILøpendeUtbetalingListe &&
+                        trekkILøpendeUtbetalingListe.map((trekkILøpendeUtbetaling, indeks) => (
+                            <TrekkILøpendeUtbetalingListeElement
+                                key={indeks}
```

Comment:
Key bør kun være indeks dersom vi vet at indeksen ikke kan endres - altså dersom elementer ikke kan fjernes fra listen. Vet vi det her? Hvis ikke så er det lurt å bruke noe annet som vi vet at er unikt for elementet

Reply:
Den kan fjernes, men da hentes hele behandlingen på nytt og trigger en ny render av alle komponenter og de vil få en ny indeks. Så indeks vil alltid være unikt, men den vil ikke alltid være det samme. Kan evt. bruke id-en til hvert trekk i stedet.

Reply:
Høres kanskje fint ut å bruke id-en istedet 🙌